packages/manager/modules/anthos/src/dashboard/general-information/general-information.routing.js

```diff
@@ -8,13 +8,19 @@ export default /* @ngInject */ ($stateProvider) => {
     resolve: {
       breadcrumb: () => false,
 
-      goBack: /* @ngInject */ (tenant, goToTenant) => (message, type) =>
-        goToTenant(tenant, message, type),
+      goBack: /* @ngInject */ ($state, goToTenant) => (message, type) =>
+        goToTenant(message, type, $state.$current.parent.name),
```

Comment:
Can we use the state name directly here instead?

Reply:
it will use the parent state of current state.